hammers/scripts/reservation_usage_notification.py

```diff
@@ -4,131 +4,57 @@
-a:visited {{ color: #542A95; text-decoration: none }}
-a:hover {{ color: #B40057; background-color:#C4FFF9; text-decoration: underline }}
-</style>
-
-<div class="body">
-<p>Dear {{{{ vars['username'] }}}},</p>
-<br>
-
-{email_body}
-
-<br>
-<p><i>
-This is an automatic email, please <b>DO NOT</b> reply! 
-If you have any question or issue, please submit a ticket on our <a href='https://www.chameleoncloud.org/user/help/' target='_blank'>help desk</a>.
-</i></p>
-
-<br><br>
-<p>Thanks,</p>
-<p>Chameleon Team</p>
-
-</div>
-<br><br>
-'''
-
-RESERVATION_START_EMAIL_BODY = '''
-<p>We're sending this email to remind you that your lease {{ vars['leasename'] }} (ID: {{ vars['leaseid'] }}) under project {{ vars['projectname'] }} on {{ vars['site'] }} 
-will start on {{ vars['startdatetime_utc'] }} UTC / {{ vars['startdatetime_ct'] }} Central Time.</p>
-'''
-
-IDLE_RESERVATION_EMAIL_BODY = '''
-<p>We're sending this email to inform you that your lease {{ vars['leasename'] }} (ID: {{ vars['leaseid'] }}) under project {{ vars['projectname'] }} on {{ vars['site'] }} 
-has been idle for more than {{ vars['idlehours_threshold'] }} hours.</p>
-
-<p>If you don't need the reservation anymore, please delete the lease using 
-either the Chameleon <a href='https://chameleoncloud.readthedocs.io/en/latest/technical/gui.html#reservations' target='_blank'>web interface</a> 
-or <a href='https://chameleoncloud.readthedocs.io/en/latest/technical/cli.html' target='_blank'>command line interface</a> to release the resources.</p>
-'''
-
 IDLE_HOUR_THRESHOLD = 24
 
-def render_template(email_body, **kwargs):
-    ''' renders a Jinja template into HTML '''
-    templ = Environment(autoescape=select_autoescape(default_for_string=True)).from_string(EMAIL_TEMPLATE.format(email_body=email_body))
-    return templ.render(**kwargs)
- 
-def send_email(email_host, to, sender, subject=None, body=None):
-    # convert TO into list if string
-    if type(to) is not list:
-        to = to.split()
-    to_list = filter(None, to) # remove null emails
-    
-    msg = MIMEMultipart('alternative')
-    msg['From'] = sender
-    msg['Subject'] = subject
-    msg['To'] = ','.join(to)
-    msg.attach(MIMEText(body, 'html'))
-
-    # send email
-    server = smtplib.SMTP(email_host, timeout=30)
-    server.sendmail(sender, to_list, msg.as_string())
-    server.quit()
-    
+
 def get_reservations_start_next_day(db):
     advance_reservations = query.get_advance_reservations(db)
     results = []
     for obj in advance_reservations:
         start_date = obj['start_date'].replace(tzinfo=tz.gettz('UTC'))
-        nextday = datetime.today().replace(tzinfo=tz.gettz('UTC')) + timedelta(days=1)
-        nextday_start = nextday.replace(hour=0, minute=0, second=0).replace(tzinfo=tz.gettz('UTC'))
-        nextday_end = nextday.replace(hour=23, minute=59, second=59).replace(tzinfo=tz.gettz('UTC'))
+        nextday = datetime.today().replace(tzinfo=tz.gettz('UTC')) + timedelta(
```

Comment:
Are you using auto-formatters for whitespace? If so, what are the settings and should we just standardize it?

Reply:
This was all zhuo's code I think. Whitespace where? This code block is for time.